ts.js

```diff
@@ -20,6 +20,9 @@ module.exports = {
     '@typescript-eslint/explicit-function-return-type': 'off', // allow compiler to derive return type
     '@typescript-eslint/no-this-alias': 'off', // allow 'const self = this'
     'jsdoc/require-param': 'off', // do not require jsdoc for params
-    'jsdoc/require-param-type': 'off' // allow compiler to derive param type
+    'jsdoc/require-param-type': 'off', // allow compiler to derive param type
+    '@typescript-eslint/no-misused-promises': 'error',
+    '@typescript-eslint/no-floating-promises': 'error',
```

Comment:
>  '@typescript-eslint/no-floating-promises': 'error',

I think this rule is included in [standard-with-typescript](https://github.com/standard/eslint-config-standard-with-typescript/blob/master/src/index.ts#L102) which this config extends so this is redundant.